### ISSUE TITLE
Does the results page always need to link to DataGraft?

### ISSUE THREAD

**Issue by fionntan** [Data-Liberation-Front/csvlint.io#651]
Hi, 

At the bottom of the results page for any CSV it says the following:

> **Next Steps**
> Publish and transform your data using [DataGraft](https://datagraft.io), either as enhanced CSV or Linked Data.

It seems odd that this service would direct people to one particular data quality tool when so many exist. Plus DataGraft mightn't even be the best solution for this user's issues. I assume this was added as part of some old project or collaboration.

A solution could be either just removing this particular Next Steps advice OR link to a selection of good, open-source data quality tools (which may or may not include DataGraft) 

Thank you,

**Comment by Floppy** [comment]
Good call; this was definitely a sensible thing to point people at back when it was added, but technology has moved on a lot and there's more out there. We could certainly look at extending the list, I don't see why not.
